infrastructure/networked_rds/README.md

```diff
@@ -0,0 +1,148 @@
+# terraform-aws-postgres-vpc
```

Comment:
this looks like a copy paste of the README from [100Automations/terraform-aws-postgres-vpc]. It will be more useful to simply put a link to it or the relevant parts, and focus this documentation on describing the details of this food oasis infrastructure configuration -- what it does, how to use it, etc.